src/styles/components/map-explorer.styl

```diff
@@ -0,0 +1,22 @@
+.map-explorer
+  width: 100%
+  overflow: hidden
+  display: flex
+  position: relative
+  
+  .map-visuals
+    flex: 0 0 auto
+    width: 100%
+    height: 100%
+    
+    #mapVisuals
```

Comment:
CSS classes or ids aren't typically camel cased. If this id is being used in javascript, it might be a good idea to decouple the styles and the javascript usage if possible. Use a CSS class instead and use a `data` property to target the DOM element? Or a React `ref`? I'm not very familiar with Leaflet's lib, though, so you can ignore this if using an id is necessary for its API.